auth/auth-domain/src/main/java/shop/woowasap/auth/domain/Password.java

```diff
@@ -0,0 +1,29 @@
+package shop.woowasap.auth.domain;
+
+import lombok.Getter;
+import shop.woowasap.auth.domain.exception.PasswordValidateException;
+
+@Getter
+public final class Password {
```

Comment:
내부 패키지에서만 사용한다면, 접근제어자를 package-private으로 변경해서 외부 패키지에서 접근을 막는것도 좋겠네요

```suggestion
final class Password {
```

Reply:
좋은 의견입니다. [91e92f7](https://github.com/woowasap/ASAP/pull/43/commits/91e92f70f4d415890437ae025da0501411f49bfc)